Samples~/SampleGame/Assets/Scripts/Server/GameLiftServer.cs

```diff
@@ -62,31 +64,13 @@ public void TerminateGameSession(bool processEnding)
             Environment.Exit(0);
         }
 
-        try
+        if (processEnding)
         {
-            GenericOutcome outcome = GameLiftServerAPI.TerminateGameSession();
-
-            if (outcome.Success)
-            {
-                _logger.Write(":) GAME SESSION TERMINATED");
-
-                if (processEnding)
-                {
-                    ProcessEnding();
-                }
-                else
-                {
-                    ProcessReady();
-                }
-            }
-            else
-            {
-                _logger.Write(":( GAME SESSION TERMINATION FAILED. TerminateGameSession() returned " + outcome.Error.ToString());
-            }
+            ProcessEnding();
         }
-        catch (Exception e)
+        else
         {
-            _logger.Write(":( GAME SESSION TERMINATION FAILED. TerminateGameSession() exception " + Environment.NewLine + e.Message);
+            ProcessReady();
```

Comment:
1. I'm guessing we added exception handling because ProcessEnding can throw an Exception potentially? 
2. Maybe this is old code, but do you know why we're calling ProcessReady in the else block? Seems weird that TerminateGameSession has ProcessReady related code based on a boolean flag passed in. Can you check if we can separate that ProcessReady logic out based on how it's being used?

Reply:
This is indeed old code. I can't seem to figure out why we would want to start up another process when we are calling for a game termination at that specific point, especially when it has to choose between ending the current process or creating a new one, not both.

Reply:
As discussed offline we will be creating a separate story to track removing TerminateGameSession. However, this should still be under a try catch block to catch exceptions?